packages/v4/src/content/get-started/about.md

```diff
@@ -14,7 +14,7 @@ PatternFly is an open source design system created to enable consistency and usa
 Components, like buttons and alerts, can be assembled together to build applications.
 
 ### Layouts
-Layouts are generic tools that allow you to structure and organize the content on your pages.
+Layouts are containers that allow you position components on pages.
```

Comment:
This looks like a rewrite of the old copy, which existed before the PF home page copy rewrite. The copy we have now is: 

_A layout provides a fully responsive page structure that keeps your components organized and aligned regardless of screen size._ 

So maybe we can add the bit that you have? That would give us: 

_Layouts are containers that allow you to position components on pages. They provide fully responsive page structures that keep your components organized and aligned regardless of screen size._

Reply:
I didn't know about this new copy on the homepage. I've updated this PR to use it!